packages/form/src/Field/Field.tsx

```diff
@@ -0,0 +1,197 @@
+import { Slot } from "@radix-ui/react-slot";
+import { TextField } from "@telegraph/text-field";
+import { Text as TelegraphText } from "@telegraph/typography";
+import { clsx } from "clsx";
+import React from "react";
+
+import { COLOR } from "./Field.constants";
+
+type RootProps = Omit<
+  React.HTMLProps<HTMLInputElement>,
+  "value" | "onChange" | "size"
+> & {
+  disabled?: boolean;
+  error?: boolean | string;
+  value?: string;
+  onChange?: (
```

Comment:
Hm, this is surprising to see — why does this component need to care about `value` and `onChange`? Shouldn't it be the input component's concern?

Reply:
This is needed as it stores `value` and `onChange` in context so it can be used in the `Control` component.